### DISCUSSION TITLE
CMU 15-112 编程和计算机科学基础 | hackway.org

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#24] [Announcements]
# CMU 15-112 编程和计算机科学基础 | hackway.org

课程名称： Structure and Interpretation of Computer Programs

https://hackway.org/docs/cs/freshman/first/cs15112#comments

**Comment by Guojia101** [comment]
链接里面没有课程的视频吗

**Comment by Rolling-meatballs** [comment]
cs10:  https://www.bilibili.com/video/BV1fe4y1y7e8/?spm_id_from=333.337.search-card.all.click&vd_source=d9043e1938ba11a5cb105266ab877723
